test/integration/stub-schema.json

```diff
@@ -3613,6 +3613,87 @@
               }
             }
           ]
+        },
```

Comment:
A new `immunotherapy` is added to argo dictionary therefor updating stub schema to be update-to-date.